docs/use-cases/manage-fleet.md

```diff
@@ -17,38 +17,48 @@ For example, you might have 30 robots in one warehouse and 500 in another.
-      <b>5. Monitor your fleet</b>
-      <p>View the status and logs from each machine in the <a href="https://app.viam.com">Viam app</a>, gather machine sensor <a href="/data/">data</a>, and operate machines remotely from the <a href="/fleet/machines/#control"><strong>Control</strong> tab</a>. Use <a href="/registry/">modules</a> to deploy code to your fleet and manage versioning.</p>
-    </td>
-  </tr>
-</table>
+{{< table >}}
+{{< tablestep >}}
+{{<imgproc src="/use-cases/signup.png" class="fill alignright" resize="600x" style="max-width: 350px" declaredimensions=true alt="Viam app signup screen">}}
+**1. Create an account**
+
+Go to the [https://app.viam.com](Viam app) and sign up with Google, GitHub, Apple, or an email address.
+
+{{< /tablestep >}}
+{{< tablestep >}}
+{{<imgproc src="/fleet/locations.png" class="fill alignleft" resize="600x" style="max-width: 400px" declaredimensions=true alt="Two locations within an organization">}}
+**2. Create organizations and locations**
+
+Use [organizations](/fleet/organizations/), and [locations](/fleet/locations/) within them, to organize your machines into groups and manage user access.
+
+{{< /tablestep >}}
+{{< tablestep >}}
+{{<imgproc src="/fleet/app-usage/limit-access.png" class="fill alignright" resize="600x" style="max-width: 350px" declaredimensions=true alt="Limit user access">}}
+**3. Invite other users and assign permissions**
+
+Invite other users to an organization or a location to [share access](/fleet/#use-viam-for-collaboration) to the machines within it.
+Assign each user a role (owner or operator) to manage permissions.
+
+{{< /tablestep >}}
+{{< tablestep >}}
+{{<imgproc src="/fleet/app-usage/create-machine.png" class="fill alignleft" resize="600x" style="max-width: 450px" declaredimensions=true alt="Create a new machine in the Viam app">}}
+**4. Connect machines to the cloud**
+
+Users with access to a location can create and collaborate on the [machines](/fleet/machines/) within it.
+When you [install `viam-server`](/get-started/installation/) on each machine, unique keys are generated to securely connect it to the Viam app.
+Use the config builder interface to configure {{< glossary_tooltip term_id="component" text="components" >}} and {{< glossary_tooltip term_id="service" text="services" >}} for new or existing smart machines.
+You can use {{< glossary_tooltip term_id="fragment" text="fragments" >}} to streamline the process of configuring multiple similar machines.
+
+{{< /tablestep >}}
+{{< tablestep >}}
+{{<imgproc src="/use-cases/last-online.png" class="fill alignright" resize="600x" style="max-width: 100px" declaredimensions=true alt="Machine last online status indicator in the Viam app.">}}
+**5. Monitor your fleet**
+
+View the status and logs from each machine in the [https://app.viam.com](Viam app), gather machine sensor [data](/data/), and operate machines remotely from the [**Control** tab](/fleet/machines/#control).
```

Comment:
Nit: what does "gather machine sensor data" mean over "view the status and logs in the Viam app" ?

Maybe something like:

```suggestion
In the [Viam app](https://app.viam.com), you can:
- Monitor the status of each machine from its [**Logs** tab](/fleet/machines/#logs).
- View any data captured by your fleet from the [**Data** tab](/data/).
- Operate machines remotely from the [**Control** tab](/fleet/machines/#control).
```